docs/conf.py

```diff
@@ -0,0 +1,50 @@
+# Configuration file for the Sphinx documentation builder.
+#
+# For the full list of built-in configuration values, see the documentation:
+# https://www.sphinx-doc.org/en/master/usage/configuration.html
+
+# -- Project information -----------------------------------------------------
+# https://www.sphinx-doc.org/en/master/usage/configuration.html#project-information
+
+
+# The master toctree document.
+source_suffix = '.rst'
+master_doc = 'index'
+
+project = 'Karoo GP'
+copyright = '2022, Kai Staats, Grant Hawkins and Ezio Melotti'
+author = 'Kai Staats, Grant Hawkins and Ezio Melotti'
+release = '3.0'
```

Comment:
https://github.com/python/cpython/blob/main/Doc/conf.py#L36-L47
Copy these so we don't have to update the release and copyright year every time.